packages/model/src/calendar.ts

```diff
@@ -47,6 +47,27 @@ export const googleLegacyCalendarColor = [
 
 export type GoogleLegacyCalendarColor =
   typeof googleLegacyCalendarColor[number];
+
+export interface CalendarDataObject {
+  id: string;
+  googleCalendarId: string;
+  color: GoogleLegacyCalendarColor;
+  name: string;
+  syncToken?: string | null;
```

Comment:
ok first of all the `CalendarDataObject` is what our data-provider returns so technically we can say it returns whatever we feel like, undefined or null, however it might be pointless to do that which is why I will remove all these nulls.

Second - the type you are pointing as reference is actually a type that doesn't really apply anywhere - I am going to remove it's export. One that matters is the `RestCalendar` which is the type of data we get from Squidex using their REST API - we only use their REST API for creating and updating data. Why is that a big difference? Well it's because the Rest wrapper adds another meaning to nullable and optional values, but as I can tell now it does it incorrectly, to make things more fun.

The truth is we should sort out the `Rest` wrapper - I will do that in another PR though.